### DISCUSSION TITLE
Anyone else having trouble downloading from redgifs?

### DISCUSSION THREAD

**Discussion by ymgenesis** [Serene-Arc/bulk-downloader-for-reddit#333] [Q&A] [answered]
Using the following:
`python3 -m bdfr download ~/Desktop/ --verbose --search-existing --no-dupes --subreddit redgifs --sort top --limit 20 --time all`

It downloads a bunch of .txts and a couple of other images, but fails in all attempts to download video from redgifs. One example of the many:

`[2021-05-02 19:42:41,442 - bdfr.downloader - DEBUG] - Attempting to download submission gfvw9v
[2021-05-02 19:42:41,444 - bdfr.downloader - DEBUG] - Using Redgifs with url https://redgifs.com/watch/foolishforkedabyssiniancat
[2021-05-02 19:42:41,580 - bdfr.downloader - ERROR] - Site Redgifs failed to download submission gfvw9v: Could not read the page source`

when visiting the redgifs url (NSFW), it loads and plays correctly. 

Wasn't sure if I needed to open an issue or a discussion.

**Comment by Serene-Arc** [comment, marked as answer]
This has been fixed. Redgifs changed their website so that our old algorithm for scraping it no longer worked. There's a fix in the development branch already and it will be in a release very soon.
